go.mod

```diff
@@ -251,3 +250,6 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.17.1 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+// remove this when https://github.com/mdlayher/ndp/pull/32 gets merged
+replace github.com/mdlayher/ndp => github.com/xliuxu/ndp v0.0.0-20240926134643-8cf547505092
```

Comment:
I assume that the ndp patch is necessary for this change?
I see that you didn't hear back from the maintainer on this. If you can find him in the K8s Slack workspace, it may be worth sending him a DM.

Reply:
I will try to ping Matt.

Reply:
Received no response from Matt in the K8s workspace. I'm not sure if he is still active there. Can we create a fork for the Antrea project for now?

Reply:
Yes I will create a fork, please submit your PR against it

Reply:
https://github.com/antrea-io/ndp

Reply:
created PR https://github.com/antrea-io/ndp/pull/1.